libs/sdk-ui-charts/src/highcharts/chartTypes/bubbleChart/bubbleConfiguration.ts

```diff
@@ -31,9 +42,7 @@ export function getBubbleConfiguration(
                 },
                 states: {
                     hover: {
-                        marker: {
-                            enabled: false,
-                        },
+                        enabled: false,
```

Comment:
So it did not work before? Or why this change?

Reply:
original code is now deprecated and not in HCH typings
https://api.highcharts.com/highcharts/plotOptions.bubble.states.hover.marker
My change was wrong, I have fixed/removed this node completely now. And it seems to behave same as before thanks plotOptions.bubble.marker.states.hover config above